app/main.py

```diff
@@ -2,0 +1,42 @@
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        if self.health >= 0:
+            Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        if self.hidden is True:
```

Comment:
Using `is True` in the condition `if self.hidden is True:` is unnecessary. You can simply write `if self.hidden:`.